### PR TITLE
Power Outage Tracker Capstone

### PR DESCRIPTION
**Pull request by aamir-asaram**
**In this project, I've implemented the following:**
- Fetch and display power outage schedule data from the [Eskom Calendar API](https://eskom-calendar-api.shuttleapp.rs/)
### **Home Page**
- Display a nav bar with a functional back button allowing users to navigate back from a details page as well as a color toggle.
- Display all zones and sub-zones in a style matching the [Behance design](https://www.behance.net/sakwadesignstudio)
- Add an at-a-glance section to indicate the current stage
- Add a drop-down list allowing users to filter schedules by zone
### **Details Page**
- Display a ten-day power outage schedule for the selected zone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,148 @@
...
+
+I would like to acknowledge the creator of the design this web application is based on - [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md]: - [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @aamir-asaram, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- All tests are passing ✅ 
- Great project design implementation ✅   
- Cool theme-colour changing effects ✅ 
- Great project demo presentation ✅ 
- Deployed site ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (aamir-asaram) [reply README.md]: I didn't notice that, thanks a lot for pointing it out @shella12 :smile:

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**